acceptancetests/jujupy/client.py

```diff
@@ -563,9 +563,12 @@ def describe_substrate(env):
 def get_stripped_version_number(version_string):
     return get_version_string_parts(version_string)[0]
 
-
 def get_version_string_parts(version_string):
     # strip the series and arch from the built version.
+    # Note:
```

Comment:
When running the `assess_upgrade` test, this func gets called with **both** a bytes version of `version_string` as well as a string one.